core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -54,12 +54,20 @@ describe('Topic and Story viewer functionality', function() {
       objective: 'The goal is to check story viewer functionality.',
       language: 'English'
     };
+
+    var exitTutorial = false;
```

Comment:
Please avoid starting with a verb for variable names.

Reply:
Removed the variable and updated its usage in the function with i === 1.